@types/environment.d.ts

```diff
@@ -1,10 +1 @@
-export {};
-
-declare global {
-  namespace NodeJS {
-    interface ProcessEnv {
-      PORT: string;
-      NODE_ENV: string;
-    }
-  }
-}
+type Env = 'HOST' | 'NODE_ENV' | 'PORT';
```

Comment:
The `Env` variable is the environment keys we want to use in our app.

Reply:
what if an env is optional?

Reply:
Anyway, we should add it here. Here we just define what envs we are expecting.